### PR TITLE
toBoolean function

### PR DESCRIPTION
**Pull request by DvirDukhan**
This PR introduces `toBoolean` function support

### CODE REVIEW

File: tests/tck/features/expressions/typeConversion/TypeConversion1.feature [hunk 1]
```diff
@@ -101,5 +96,5 @@ Feature: TypeConversion1 - To Boolean
       | invalid |
       | []      |
       | {}      |
-      | 1       |
+      # | 1       |
```
Comment (AviAvni) [inline tests/tck/features/expressions/typeConversion/TypeConversion1.feature:99]: take a look at https://github.com/opencypher/openCypher/blob/master/tck/features/expressions/typeConversion/TypeConversion1.feature look like this file was updated

**Review by AviAvni — APPROVED** [review]

**Comment by swilly22** [comment]
@DvirDukhan @AviAvni 
Do we need to update documentation ?

**Comment by DvirDukhan** [comment]
@swilly22 https://github.com/RedisGraph/RedisGraph/pull/2507

**Comment by swilly22** [comment]
@DvirDukhan do we think this is enough documentation?
do we need to document each function individually ?

**Pull request merged**